Composer/packages/lib/indexers/__tests__/jsonSchemafileIndexer.test.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { jsonSchemaFileIndexer } from '../src/jsonSchemaFileIndexer';
+import { FileInfo } from '../src/type';
+
+const { index } = jsonSchemaFileIndexer;
+
+describe('jsonSchemaFileIndexer', () => {
+  describe('index', () => {
+    it('should return json schema files', () => {
+      const input: FileInfo[] = [
```

Comment:
nit: Maybe move this to outside and reuse it in both of the cases?

Reply:
The schemas are different in the different cases - one to ensure we include those that are schemas, one to ensure we exclude others.  I guess I could do it in one test, but it felt like 2 logical cases.